--- ../src-base/minecraft/net/minecraft/client/gui/ServerListEntryNormal.java
+++ ../src-work/minecraft/net/minecraft/client/gui/ServerListEntryNormal.java
@@ -69,12 +69,12 @@
                     catch (UnknownHostException unknownhostexception)
                     {
                         ServerListEntryNormal.this.field_148301_e.field_78844_e = -1L;
-                        ServerListEntryNormal.this.field_148301_e.field_78843_d = EnumChatFormatting.DARK_RED + "Can\'t resolve hostname";
+                        ServerListEntryNormal.this.field_148301_e.field_78843_d = EnumChatFormatting.DARK_RED + net.minecraft.client.resources.I18n.func_135052_a("adv.connection.error.hostname");
                     }
                     catch (Exception exception)
                     {
                         ServerListEntryNormal.this.field_148301_e.field_78844_e = -1L;
-                        ServerListEntryNormal.this.field_148301_e.field_78843_d = EnumChatFormatting.DARK_RED + "Can\'t connect to server.";
+                        ServerListEntryNormal.this.field_148301_e.field_78843_d = EnumChatFormatting.DARK_RED + net.minecraft.client.resources.I18n.func_135052_a("adv.connection.error.other");
                     }
                 }
             });
@@ -84,7 +84,7 @@
         boolean flag2 = this.field_148301_e.field_82821_f < 47;
         boolean flag3 = flag1 || flag2;
         this.field_148300_d.field_71466_p.func_78276_b(this.field_148301_e.field_78847_a, p_180790_2_ + 32 + 3, p_180790_3_ + 1, 16777215);
-        List list = this.field_148300_d.field_71466_p.func_78271_c(this.field_148301_e.field_78843_d, p_180790_4_ - 32 - 2);
+        List list = this.field_148300_d.field_71466_p.func_78271_c(net.minecraftforge.fml.client.FMLClientHandler.instance().fixDescription(this.field_148301_e.field_78843_d), p_180790_4_ - 48 - 2);
 
         for (int l1 = 0; l1 < Math.min(list.size(), 2); ++l1)
         {
@@ -102,7 +102,7 @@
         if (flag3)
         {
             j2 = 5;
-            s1 = flag1 ? "Client out of date!" : "Server out of date!";
+            s1 = flag1 ? net.minecraft.client.resources.I18n.func_135052_a("adv.connection.old.client") : net.minecraft.client.resources.I18n.func_135052_a("adv.connection.old.server");
             s = this.field_148301_e.field_147412_i;
         }
         else if (this.field_148301_e.field_78841_f && this.field_148301_e.field_78844_e != -2L)
@@ -134,11 +134,11 @@
 
             if (this.field_148301_e.field_78844_e < 0L)
             {
-                s1 = "(no connection)";
+                s1 = net.minecraft.client.resources.I18n.func_135052_a("adv.connection.noConnection");
             }
             else
             {
-                s1 = this.field_148301_e.field_78844_e + "ms";
+                s1 = this.field_148301_e.field_78844_e + net.minecraft.client.resources.I18n.func_135052_a("adv.measure.millisecond");
                 s = this.field_148301_e.field_147412_i;
             }
         }
@@ -152,7 +152,7 @@
                 j2 = 8 - j2;
             }
 
-            s1 = "Pinging...";
+            s1 = net.minecraft.client.resources.I18n.func_135052_a("adv.connection.pinging");
         }
 
         GlStateManager.func_179131_c(1.0F, 1.0F, 1.0F, 1.0F);
@@ -178,6 +178,11 @@
         int k2 = p_180790_6_ - p_180790_2_;
         int l2 = p_180790_7_ - p_180790_3_;
 
+        String tooltip = net.minecraftforge.fml.client.FMLClientHandler.instance().enhanceServerListEntry(this, this.field_148301_e, p_180790_2_, p_180790_4_, p_180790_3_, k2, l2);
+        if (tooltip != null)
+        {
+            this.field_148303_c.func_146793_a(tooltip);
+        } else
         if (k2 >= p_180790_4_ - 15 && k2 <= p_180790_4_ - 5 && l2 >= 0 && l2 <= 8)
         {
             this.field_148303_c.func_146793_a(s1);
