--- ../src-base/minecraft/net/minecraft/client/network/OldServerPinger.java
+++ ../src-work/minecraft/net/minecraft/client/network/OldServerPinger.java
@@ -58,7 +58,7 @@
         ServerAddress serveraddress = ServerAddress.func_78860_a(p_147224_1_.field_78845_b);
         final NetworkManager networkmanager = NetworkManager.func_150726_a(InetAddress.getByName(serveraddress.func_78861_a()), serveraddress.func_78864_b());
         this.field_147229_c.add(networkmanager);
-        p_147224_1_.field_78843_d = "Pinging...";
+        p_147224_1_.field_78843_d = net.minecraft.client.resources.I18n.func_135052_a("adv.connection.pinging");
         p_147224_1_.field_78844_e = -1L;
         p_147224_1_.field_147412_i = null;
         networkmanager.func_150719_a(new INetHandlerStatusClient()
@@ -148,6 +148,7 @@
                     p_147224_1_.func_147407_a((String)null);
                 }
 
+                net.minecraftforge.fml.client.FMLClientHandler.instance().bindServerListData(p_147224_1_, serverstatusresponse);
                 this.field_175092_e = Minecraft.func_71386_F();
                 networkmanager.func_179290_a(new C01PacketPing(this.field_175092_e));
                 this.field_147403_d = true;
@@ -164,7 +165,7 @@
                 if (!this.field_147403_d)
                 {
                     OldServerPinger.field_147228_b.error("Can\'t ping " + p_147224_1_.field_78845_b + ": " + p_147231_1_.func_150260_c());
-                    p_147224_1_.field_78843_d = EnumChatFormatting.DARK_RED + "Can\'t connect to server.";
+                    p_147224_1_.field_78843_d = EnumChatFormatting.DARK_RED + net.minecraft.client.resources.I18n.func_135052_a("adv.connection.error.other");
                     p_147224_1_.field_78846_c = "";
                     OldServerPinger.this.func_147225_b(p_147224_1_);
                 }
